src/app/cli/src/services/accounts/account_service.rs

```diff
@@ -120,17 +120,20 @@ impl AccountService {
         RelatedAccountIndication::new(target_account)
     }
 
-    fn find_account_info_impl(&self, account_name: &String) -> Option<AccountInfo> {
+    fn find_account_info_impl(&self, account_name: &AccountName) -> Option<AccountInfo> {
         // The account might be predefined in the configuration
-        self.predefined_accounts.get(account_name).cloned()
+        self.predefined_accounts.get(account_name.as_str()).cloned()
```

Comment:
If `predefined_accounts` was keyed by `AccountName` instead of `String` would this avoid needing to do any `to_lowercase` conversions? i.e. it would "just work"?

Reply:
definitely.
changed